packages/react-native/React/Views/RCTCursor.m

```diff
@@ -0,0 +1,233 @@
+      break;
+    case RCTCursorCrosshair:
+      resolvedCursor = [NSCursor crosshairCursor];
+      break;
+    case RCTCursorDefault:
+      resolvedCursor = [NSCursor arrowCursor];
+      break;
+    case RCTCursorEResize:
+#ifdef __MAC_OS_15_SDK_AVAILABLE
+      if (@available(macOS 15.0, *)) {
+        resolvedCursor = [NSCursor frameResizeCursorFromPosition:NSCursorFrameResizePositionLeft
+                                                    inDirections:NSCursorFrameResizeDirectionsOutward];
+      } else {
+        resolvedCursor = [NSCursor resizeRightCursor];
+      }
+#else
+      resolvedCursor = [NSCursor resizeRightCursor];
+#endif // __MAC_OS_15_SDK_AVAILABLE
+      break;
+    case RCTCursorEWResize:
+#ifdef __MAC_OS_15_SDK_AVAILABLE
+      if (@available(macOS 15.0, *)) {
+        resolvedCursor = [NSCursor frameResizeCursorFromPosition:NSCursorFrameResizePositionLeft
+                                                    inDirections:NSCursorFrameResizeDirectionsAll];
+      }
+#endif // __MAC_OS_15_SDK_AVAILABLE
+      break;
+    case RCTCursorGrab:
+      resolvedCursor = [NSCursor openHandCursor];
+      break;
+    case RCTCursorGrabbing:
+      resolvedCursor = [NSCursor closedHandCursor];
+      break;
+    case RCTCursorHelp:
+      // Not supported
+      break;
+    case RCTCursorMove:
+      // Not supported
+      break;
+    case RCTCursorNEResize:
+#ifdef __MAC_OS_15_SDK_AVAILABLE
+      if (@available(macOS 15.0, *)) {
+        resolvedCursor = [NSCursor frameResizeCursorFromPosition:NSCursorFrameResizePositionTopRight
+                                                    inDirections:NSCursorFrameResizeDirectionsOutward];
+      }
+      break;
+#endif // __MAC_OS_15_SDK_AVAILABLE
+    case RCTCursorNESWResize:
+#ifdef __MAC_OS_15_SDK_AVAILABLE
+      if (@available(macOS 15.0, *)) {
+        resolvedCursor = [NSCursor frameResizeCursorFromPosition:NSCursorFrameResizePositionTopRight
+                                                    inDirections:NSCursorFrameResizeDirectionsAll];
+      }
+#endif // __MAC_OS_15_SDK_AVAILABLE
+      break;
+    case RCTCursorNResize:
+      resolvedCursor = [NSCursor resizeUpCursor];
+#ifdef __MAC_OS_15_SDK_AVAILABLE
+      if (@available(macOS 15.0, *)) {
+        resolvedCursor = [NSCursor frameResizeCursorFromPosition:NSCursorFrameResizePositionTop
+                                                    inDirections:NSCursorFrameResizeDirectionsOutward];
+      } else {
+        resolvedCursor = [NSCursor resizeUpCursor];
+      }
+#else
+      resolvedCursor = [NSCursor resizeUpCursor];
+#endif // __MAC_OS_15_SDK_AVAILABLE
+      break;
+    case RCTCursorNSResize:
+#ifdef __MAC_OS_15_SDK_AVAILABLE
+      if (@available(macOS 15.0, *)) {
+        resolvedCursor = [NSCursor frameResizeCursorFromPosition:NSCursorFrameResizePositionTop
+                                                    inDirections:NSCursorFrameResizeDirectionsAll];
+      }
```

Comment:
Missing fallback?

Reply:
No fallback available, short of shipping our own asset or looking through webkit code to see where there Asset is.